src/rules/banTsIgnoreRule.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2013 Palantir Technologies, Inc.
```

Comment:
Will you change the dates to 2018 in this file and the examples file?

Reply:
Done.